src/actions.js

```diff
@@ -0,0 +1,35 @@
+// action types
+export const ActionTypes = {
+  UNDO: '@@redux-undo/UNDO',
+  REDO: '@@redux-undo/REDO',
+  JUMP_TO_FUTURE: '@@redux-undo/JUMP_TO_FUTURE',
+  JUMP_TO_PAST: '@@redux-undo/JUMP_TO_PAST',
+  JUMP: '@@redux-undo/JUMP',
+  CLEAR_HISTORY: '@@redux-undo/CLEAR_HISTORY'
+}
+// /action types
```

Comment:
I think we could drop a lot of these comments now. We all know what it is right? And possible new contributors would either be familiair with redux or should be :p